Library/ViewModels/PostCampaignPledgeRewardsSummaryViewModel.swift

```diff
@@ -189,10 +189,10 @@ private func items(
   }
   var items = [headerItem] + rewardItems
 
-  if featureNoShippingAtCheckout() == false {
-    // MARK: Shipping
+  if let shipping = data.shipping {
+    if shipping.total > 0 {
```

Comment:
nit: This is a little nicer to read if you do `if let shipping = data.shipping, shipping.total > 0 {` instead of nesting two `if`s.